presto-docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -293,3 +293,15 @@ Property Name                                    Description
 ``elasticsearch.aws.secret-key``                 AWS secret key to use to connect to the Elasticsearch domain.
 ================================================ ==================================================================
 
+X-Pack/Basic Authentication
```

Comment:
This should say just "password authentication". "X-Pack" is an Elasticsearch extension framework and, as such, it's an implementation detail of Elasticsearch.